src/System.Diagnostics.Tools/tests/System/Diagnostics/CodeAnalysis/SuppressMessageAttributeTests.cs

```diff
@@ -0,0 +1,46 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using Xunit;
+
+namespace System.Diagnostics.CodeAnalysis.Tests
+{
+    public static class SuppressMessageAttributeTests
+    {
+        [Fact]
+        public static void TestConstructorBasic()
```

Comment:
It should not right, the class is also static, hence it can start the tests in separate threads. Or is it that, xunit's design is such, that it wont run tests in parallel if they are static?